pkg/republisher/uuid_republisher_test.go

```diff
@@ -198,6 +198,116 @@ func TestFoundInNoneAndNotFoundInDocStore_Ok(t *testing.T) {
 	assert.True(t, strings.HasSuffix(errs[0].Error(), "wasn't found in any of the native-store's collections and it's not an ImageSet"))
 }
 
+func TestUUIDInUCAndMethodeRepublishedInUC(t *testing.T) {
+	mockedUCRepublisher := new(mockUCRepublisher)
+	mockedDocStoreClient := new(mockDocStoreClient)
+	msg := OKMsg{
+		uuid:                     "64bc4319-cd22-43e9-8b12-358622d7a5ba",
+		tid:                      "prefix1tid_123",
+		collectionName:           "universal-content",
+		collectionOriginSystemID: "cct",
+		sizeBytes:                1024,
+		notifierAppName:          "cms-notifier",
+	}
+
+	msg2 := OKMsg{
+		uuid:                     "b3ec9282-1073-46ad-9d44-144dad7fe956",
+		tid:                      "prefix1",
+		collectionName:           "pac-metadata",
+		collectionOriginSystemID: "http://cmdb.ft.com/systems/pac",
+		sizeBytes:                1024,
+		notifierAppName:          "cmsMetadataNotifier",
+	}
+
+	msg3 := OKMsg{
+		uuid:                     "b3ec9282-1073-46ad-9d44-144dad7fe956",
+		tid:                      "prefix1",
+		collectionName:           "methode",
+		collectionOriginSystemID: "methode-web-pub",
+		sizeBytes:                1024,
+		notifierAppName:          "cms-notifier",
+	}
+
+	var nilMsg *OKMsg
+	mockedUCRepublisher.On("RepublishUUIDFromCollection", "64bc4319-cd22-43e9-8b12-358622d7a5ba", mock.MatchedBy(func(tid string) bool { return strings.HasPrefix(tid, "prefix1") }), testCollections["methode"]).Return(&msg3, false, nil)
```

Comment:
I see that you've added a new message but the value remains `false`. Shouldn't it be set to `true`?
```suggestion
	mockedUCRepublisher.On("RepublishUUIDFromCollection", "64bc4319-cd22-43e9-8b12-358622d7a5ba", mock.MatchedBy(func(tid string) bool { return strings.HasPrefix(tid, "prefix1") }), testCollections["methode"]).Return(&msg3, true, nil)
```

Reply:
right, it has been fixed